_includes/project-team-loop.html

```diff
@@ -7,14 +8,60 @@
   $('[data-toggle="tooltip"]').tooltip()
 })</script>
 
-{% assign teammembers = site.data.ph_authors | sort: "sortname" %}
-{% for member in teammembers %}
-  {% if member.team == true %}
-    {% capture membername %}{{ member.name }}{% endcapture %}
-    {% include contact-info.html name=membername %}
-  {% endif %}
+
+<h3>Programming Historian in English</h3> 
+{% assign englishmembers = site.data.ph_authors | sort: "sortname" %}
```

Comment:
This looks good. 

You can rewrite each of the blocks to filter the team members more directly like so:

```
{% assign englishmembers = site.data.ph_authors | where_exp: "member", "member.team_roles contains 'editorial'" | where_exp: "member", "member.team == true" | sort: "sortname" %}
{% for member in englishmembers %}
{% include contact-info.html name=member.name %}
{% endfor %}
```

This filters twice: first, checking the `team_roles` contains the right team, and then next filtering to check that `team` is `true`.

Also, note that you don't need to use the `capture` tag - you can directly pass `member.name` to the `contact-info.html` include.

Doing this reduces the number of `if` and `for` statements so it's easier to see what the code is doing.

Reply:
Thanks. Are these changes I have to add, or have you added them? Sorry, still learning the technical side of things on github.

Reply:
You need to add them. You can use the code I put in this comment, but you need to add it for each of the teams, adjusting it appropriately each time.

Reply:
Interestingly, looking at the ToC, I think that you will need to express the `<h3>` header using Markdown in order to get it to be recognized by JEkyll. So replace

```html
<h3>Programming Historian in English</h3>
```

with

```md
### Programming Historian in English
```